cmd/bee/cmd/start.go

```diff
@@ -259,10 +269,12 @@ func (c *command) initStartCmd() (err error) {
 
 					// If shutdown function is blocking too long,
 					// allow process termination by receiving another signal.
+					shutdownTimeout := time.Second * 5
 					select {
-					case <-sysInterruptChannel:
-						logger.Debug("received interrupt signal")
+					case <-time.After(shutdownTimeout):
```

Comment:
This is not correct. Adding a hard-coded timeout is not the right thing to do. There could be valid cases when the node takes more than 5 secs to stop.

As the comment said, the correct way to handle this is to check for a second interrupt signal to force stop if user does not have the patience. I am not sure if this can be done using context cancellation.